### DISCUSSION TITLE
Free navigation simulation

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#154] [Q&A] [answered]
As a quick demo, Cruiser free was installed this for an interested motorcyclist.
With an unlocked navigation simulation function, the demo could have been even more enticing.

**Comment by devemux86** [comment]
Hi @0709wiwiwi,

_The subscription has a free 7-day trial period._

Do you mean the navigation simulation to be available to all for free?

**Comment by devemux86** [comment]
@0709wiwiwi thanks, this is an interesting idea!

Navigation simulation can be available for free in the next Cruiser.

**Comment by devemux86** [comment, marked as answer]
Navigation simulation is available for **free** in [Cruiser 3.0.19](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
